staging/src/k8s.io/apiserver/pkg/util/peerproxy/gv_exclusion_manager.go

```diff
@@ -0,0 +1,550 @@
+/*
```

Comment:
Is this a rename of `exclusion_filter`?  Can we have one commit that only does the rename so that it's easier to review the actual diff?